PyBall/models/draft/prospect.py

```diff
@@ -0,0 +1,21 @@
+from PyBall.models.draft.home import Home
+from PyBall.models.draft.school import School
+from PyBall.models.draft.person import Person
+
+
+class Prospect:
```

Comment:
Prospect and Pick seem to be identical. Should they be combined?

Reply:
Im not against it, but Im not sure what to name name the more generalized class.